charts.py

```diff
@@ -527,7 +527,11 @@ def month_completion(dframe, z_values='count',colorscale='Portland'):
 ################################################################################
 ################################################################################
 # ---- opened vs completion gap (backlog) ---- #     
-def linechart_gap_refactor_test(dframe, frequency, id_column='wo_id', completion_column='date_completed'):
```

Comment:
is a better name. more descriptive and sets up easier if chart becomes instance of a chars class or line charts class.